src/Controller/TestEmailController.php

```
@@ -0,0 +1,115 @@
+  /**
+   * Handles test email sending via AJAX.
+   *
+   * @return \Symfony\Component\HttpFoundation\JsonResponse
+   *   JSON response with success/error status.
+   */
+  public function sendTestEmail() {
+    try {
+      // Get config using the proper method
+      $config = \Drupal::config('bento_sdk.settings');
+      $publishable_key = $config->get('publishable_key');
+      $site_uuid = $config->get('site_uuid');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$site_uuid'. (Unused Code Rules)

(UnusedLocalVariable)